src/Compilers/Core/Portable/Symbols/Attributes/CommonTypeEarlyWellKnownAttributeData.cs

```diff
@@ -113,5 +113,23 @@ public bool HasCodeAnalysisEmbeddedAttribute
             }
         }
         #endregion
+
+        #region InterpolatedStringHandlerAttribute
+        private bool _hasInterpolatedStringHandlerAttribute;
+        public bool HasInterpolatedStringHandlerAttribute
```

Comment:
>HasInterpolatedStringHandlerAttribute [](http://example.com/codeflow?start=20&length=37)

What is the reason for putting this into EarlyWellKnownAttributeData?

Reply:
This is covered in the actual decode code, and has a corresponding test to make sure we bind to the correct attribute constructor.